LICENSE

```diff
@@ -22,39 +22,19 @@ If not, see [GNU licenses](http://www.gnu.org/licenses/).
 
 The following files and directories are exceptions:
 
-* app/views/unattended/ztp/provision.erb is under 2-clause BSD license
-* config/locales/ and public/blank.html are MIT licensed
-* lib/tasks/convert.rake is under Revised BSD license
-* extras/noVNC/websockify is LGPL v3 licensed
-* vendor/assets/fonts/ is ASL 2.0 licensed
-* vendor/assets/javascripts/ace/ is under Revised BSD license
-* vendor/assets/javascripts/noVNC/ is LGPL v3, GPL v2+, MIT and BSD
-* vendor/assets/javascripts/diff.js is under Revised BSD license
-* vendor/assets/javascripts/jquery.cookie.js is GNU GPL v2/MIT licensed
-* vendor/assets/javascripts/jquery.dataTables.min.js is GNU GPL v2/BSD
-* vendor/assets/javascripts/jquery.flot.axislabels.js is GNU GPL v3/MIT licensed
-* vendor/assets/javascripts/jquery.jeditable.js is MIT licensed
-* vendor/assets/javascripts/jquery.jnotify.js is ASL 2.0 licensed
-* vendor/assets/javascripts/jstz.js is MIT licensed
-
-Holders for code above are:
-
-  (c) 2013, Juniper Networks
-  (c) 2008, Matson Systems, Inc.
-  (c) 2010, Ajax.org B.V.
-  (c) 2005, Brad Neuberg, bkn3@columbia.edu
-  (c) 2009-2011, Kevin Decker <kpdecker@gmail.com>
-  (c) Hiroshi Ichikawa <http://gimite.net/en/>
-  (c) Martijn Pieters <mj@digicool.com>, Samuel Sieb <samuel@sieb.net>
-  (C) 1999 AT&T Laboratories Cambridge
-  (c) 1996 Widget Workshop, Inc.
-  (C) 1996 by Jef Poskanzer <jef@acme.com>
-  (c) 2010 Klaus Hartl (stilbuero.de)
-  (c) 2008-2012 Allan Jardine
-  (c) 2010 Xuan Luo
-  (c) 2006-2009 Mika Tuupola, Dylan Verheul
-  (c) 2010 Giva, Inc.
-  (c) 2012-2014 Jon Nylander
+* app/views/unattended/ztp/provision.erb is (c) 2013, Juniper Networks under 2-clause BSD license.
+* public/blank.html is (c) 2005, Brad Neuberg, bkn3@columbia.edu under MIT license.
+* lib/tasks/convert.rake is (c) 2008, Matson Systems, Inc. under Revised BSD license.
+* extras/noVNC/websockify is (c) 2011, Joel Martin under LGPL v3 license.
+* vendor/assets/fonts/ is (c) 2015, Red Hat Inc. under SIL Open Font License v1.1 and LGPL v2.1 licenses.
```

Comment:
1+2. they are the same licence and copyright, I created AllBullets to allow multi-line text areas to be hidden (https://github.com/tbrisker/AllBullets).
3. fixed
4. I'll make it 2011-2016.